### DISCUSSION TITLE
Support of RC5 ext IR protocol

### DISCUSSION THREAD

**Discussion by nicoboyfr** [arendst/Tasmota#16996] [Ideas]
Hello,

I have a Marantz amplifier that use RC5 IR protocol and also RC5 extended.
Would be nice if tasmota IR can recognize such protocol.
In details, RC5 ext is not that different than RC5, it extends the RC5 code to 128 commands.
RC5 sequence is 14 bits long, the 2 first bits are always 1 (start bits).
RC5 ext only differ by the bit1 (2nd start bit) set to 0 instead of 1.

Thanks.

**Comment by sfromis** [comment]
The IR support is not internal to the Tasmota project, all that happens in https://github.com/crankyoldgit/IRremoteESP8266 instead, where enhanced support would have to be implemented.
